tools/renovate/tasks/sdk-regenerate.json

```diff
@@ -9,6 +9,7 @@
       "postUpgradeTasks": {
         "commands": [
           "{{arg0}} install",
+          "{{arg0}} run schematics @ama-sdk/schematics:migrate -- --from={{{currentVersion}}} --to={{{newVersion}}}",
```

Comment:
Do we always have the double '--' or only in case of 'npm' ?

Reply:
It is not needed for yarn indeed but not an issue neither.
(Not sure there is a way to add `--` only for `npm`)

Reply:
ok for me :)